js/PaymentRequest/index.js

```diff
@@ -106,26 +109,33 @@ export default class PaymentRequest {
   _serializedModifierData: string;
   _details: Object;
   _options: Object;
+  _creditDebitDetails: Object;
```

Comment:
Would it make sense to name this object `_cardType`?  If that name is accurate, I think it would be clear and concise.  I might just be misunderstanding the usage of this object though.

Reply:
_creditDebitDetails includes all the details of both debit and credit cards :)